packages/vars/stories/Story.css.ts

```diff
@@ -16,6 +16,11 @@ export const local = style({
   vars: {
     [accentVar]: vars.color.pink[500],
   },
+  selectors: {
+    ['[data-theme="dark"] &']: {
+      vars: { [accentVar]: vars.color.pink[200] },
+    },
+  },
```

Comment:
Simple example of how to tweak a local CSS custom property (component-level token) for dark mode.